web/templates/invite_user_modal.hbs

```diff
@@ -32,6 +32,13 @@
             <textarea rows="2" id="invitee_emails" name="invitee_emails" class="invitee_emails" placeholder="{{t 'One or more email addresses...' }}"></textarea>
         </div>
     </div>
+    <div class="input-group new-style" id="receive-invite-notification">
```

Comment:
It would probably be better to name it as `receive-invite-acceptance-notification` to make it more clear.